### DISCUSSION TITLE
Why scaleDims=TRUE do z-score normalization for each cell instead of each feature

### DISCUSSION THREAD

**Discussion by RoseYuan** [GreenleafLab/ArchR#1935] [Questions / Documentation]
Hi all,

I'm looking at the function `addIterativeLSI()` and trying to understand what `scaleDims = TRUE` does to the feature matrix. If I understand it correctly, it calculates the z-score normalization for each row (i.e. each cell), according to the code [here](https://github.com/GreenleafLab/ArchR/blob/f6c0388bd37023400794c9ae8562ad69e3ba9fd7/R/ProjectMethods.R#L752-L758). I don't understand why one would do this. Let's say we have a matrix `x` of 3 cells x 3 features: `x <- t(matrix(c(1,2,3,1,3,3,10,11,12),nrow=3))` wouldn't it be obvious that cell 1 and cell 2 are more likely to be from the same cluster than cell 1 and cell 3? But after applying the `.rowZscores` function, the matrix becomes like this,

> -0.5773503 | -0.5773503 | 1.154701      
> -0.6757374 | -0.4730162 | 1.148754      
> -0.5773503 | -0.5773503 | 1.154701

suggesting that cell 1 and cell 3 are from the same cluster... So I'm confused why this normalization could be good.

**Comment by dannyconrad** [comment]
I just raised this concern as well at #2120 before I saw this thread. I agree this probably needs to be addressed.
